pixi/backend/api.js

```diff
@@ -23,9 +23,2797 @@ api.get('/hello-world', async (request, response) => {
   log.info('hello-world endpoint called.');
 
   response.setHeader('Content-Type', 'application/json');
-  response
-    .status(200)
-    .send(JSON.stringify({'message': 'Hello World'}, null, 2));
+  response.status(200).send(
+    JSON.stringify(
+      {
+        'message': 'Hello World',
+      },
+      null,
+      2
+    )
+  );
+});
+
+api.get('/page-experience-dummy', async (request, response) => {
```

Comment:
Do we need the dummy at all to start? Can we call the PageSpeedInsights API with `https://amp.dev` hardcoded for now?

Reply:
@caroqliu we're using the dummy API to reduce waiting times during development since the _[real one](https://www.googleapis.com/pagespeedonline/v5/runPagespeed?url=http://amp.dev/&key=AIzaSyCKKBvhpC73FqDcO-T7_4Yqdx4nQXh2sQY)_ sometimes takes up to ~10 sec to load.

Reply:
In that case, can we still set up the `url` param endpoint and delegate to PageSpeedInsights so it is available for manual testing in staging? We can still use this endpoint for development.